### PR TITLE
Create basic app with minimum functionality

### PR DESCRIPTION
**Pull request by bengigo**
Changes made in this branch:

-Setup the project with React
-Create a list structure and components
-Create a form to add new tasks to the list
-Implement functionalities of checking off and deleting single tasks using React event handlers

### CODE REVIEW

File: src/components/InputTodo.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+class InputTodo extends Component {
+  state = {
+    title: '',
+  };
+
+  onChange = (e) => {
+    this.setState({
+      title: e.target.value,
+    });
+  };
```
Comment (tarek102) [inline src/components/InputTodo.js:13]: It would be nice if you declared another name to avoid confusion about onChange function.

**Review by tarek102 — CHANGES_REQUESTED** [review]
Hello @bengigo, 

Good job you did, however there is one thing that needs to be changed 



### To Highlight 👏

- Gitflow was correctly used. ✔️
- No Linter errors ✔️
- Commits are descriptive ✔️
- PR is descriptive ✔️
- README.md is descriptive ✔️

### Required Changes ♻️

- Please remove unused files (eg. favicon.ico, logo5012.png, Navbar.js, ..etc)

File: src/components/InputTodo.js (hunk 1 above)
Comment (bengigo) [reply src/components/InputTodo.js:13]: The name is provided from a tutorial. To keep the tutorial work more manageable and cleaner, I can finish the tutorial, then edit function names.

**Comment by bengigo** [comment]
Thank you for pointing that out. I have removed unused files.

**Review by tarek102 — APPROVED** [review]
Hello [@bengigo ](https://github.com/bengigo) ,

Your project is complete! There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏

Congratulations! 🎉 🎉 🎉

### Highlights
Professional Readme File ✔✔
Linters are setup correctly ✔✔
Calculator is working normally ✔✔

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**